specs/language/declarations.tex

```diff
@@ -96,10 +96,14 @@
 assignment-expression in an initializer-list and performing a depth-first
 traversal accessing each subobject of the assignment-expression.
 
-\p An initializer-list is a valid initializer if for each element \(E_n\) in the
-target object's flattened ordering there is a corresponding initializer \(I_n\)
-in the flattened initializer sequence which can be implicitly converted to the
-element's type.
+\p If the target object is an array of unknown size, the object is assumed to
+have \(m\) possible elements during parsing, where \(m>0\).
+
+\p An initializer-list is a valid initializer if for each element \(E_n\mod{}m\)
```

Comment:
Is the 'mod m' meant to be part of the subscript? If so you need to change this to 'E_{n\mod{}m}'. If not, then I'm not sure what you are saying.